src/python/pants/init/import_util.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2025 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import annotations
+
+import importlib.metadata
+import re
+from collections.abc import Generator
+from importlib.metadata import Distribution
+
+from packaging.requirements import Requirement
+from packaging.version import InvalidVersion, Version
+
+
+def normalize_name(name: str) -> str:
```

Comment:
See above for the existence of this functionality in `packaging.utils`. Looks like in this PR it's also duplicated?

Reply:
Will fix.